modules/storages/config/locales/en.yml

```
@@ -251,22 +278,31 @@ en:
         oidc_cant_refresh_token: There was an error while trying to check your access to the storage. Please check the server logs for further information.
         oidc_non_oidc_user: The current user, while provisioned, wasn't provisioned by an OpenID Connect (OIDC) Identity Provider. Please re-run the check with an OIDC provisioned user.
         oidc_non_provisioned_user: The current user isn't provided by an OpenID Connect Identity Provider. Please re-run the check with a provided user.
-        placeholder: Check your connection against the server.
-        subtitle: Connection validation
         tenant_id_wrong: The configured directory (tenant) id is invalid. Please check the configuration.
         unexpected_content:
           nextcloud: Unexpected content found in the managed group folder.
           one_drive: Unexpected content found in the drive.
         unknown_error: The connection could not be validated. An unknown error occurred. Please check the server logs for further information.
         userless_access_denied: The configured app password is invalid.
       label_error: Error
+      label_failed: Failed
       label_healthy: Healthy
+      label_passed: Passed
       label_pending: Pending
+      label_skipped: Skipped
       label_warning: Warning
+      no_report: No report available
+      no_report_description: "Run the tests now for a full health status report for this file storage."
```

Translation consistency:

```suggestion
      no_report_description: "Run the checks now for a full health status report for this file storage."
```